deployments/deployment.yaml

```diff
@@ -10,13 +10,14 @@ spec:
   selector:
     matchLabels:
       app: totally-not-peirates
-  strategy: {}
+  strategy:
+    type: RollingUpdate
   template:
     metadata:
       labels:
         app: totally-not-peirates
     spec:
       containers:
-      - name: peirates
-        image: ghcr.io/devsecfranklin/peirates
-        imagePullPolicy: Always
+        - name: peirates
```

Comment:
nit: Indentation inconsistent with Kubernetes style and the deployment-dev.yaml. (Although it is valid.)

Reply:
Hm, I am using a vscode plugin for this. "Prettier". Also have a "YAML" plug in that does the same indentation. 

Once issue here is that the syntax checker wants me to put resource limits on the container. 

Without the limits in place
![before](https://github.com/inguardians/peirates/assets/2730246/258c1c8d-5d8a-4286-a5d9-eb9d0c798c11)

and with some arbitrary limits just to show the diff
![after](https://github.com/inguardians/peirates/assets/2730246/a1305f17-7537-4bbf-b147-215f0571d253)

Reply:
Not exactly sure how to deal with this.

Reply:
I got this one.